spyder/plugins/completion/kite/widgets/install.py

```diff
@@ -0,0 +1,389 @@
+        general_layout.addLayout(action_layout)
+        self.setLayout(general_layout)
+
+        # Signals
+        install_button.clicked.connect(self.sig_install_button_clicked)
+        dismiss_button.clicked.connect(self.sig_dismiss_button_clicked)
+
+
+class HoverEventFilter(QObject):
+    """QObject to handle event filtering."""
+    # Signal to trigger on a HoverEnter event
+    sig_hover_enter = Signal()
+    # Signal to trigger on a HoverLeave event
+    sig_hover_leave = Signal()
+
+    def eventFilter(self, widget, event):
+        """Reimplemented Qt method."""
+        if event.type() == QEvent.HoverEnter:
+            self.sig_hover_enter.emit()
+        elif event.type() == QEvent.HoverLeave:
+            self.sig_hover_leave.emit()
+
+        return super(HoverEventFilter, self).eventFilter(widget, event)
+
+
+class KiteInstallation(QWidget):
+    """Kite progress installation widget."""
+
+    # Signal to check clicks on the ok button
+    sig_ok_button_clicked = Signal()
+    # Signal to check clicks on the cancel button
+    sig_cancel_button_clicked = Signal()
+
+    def __init__(self, parent):
+        super(KiteInstallation, self).__init__(parent)
+
+        # Left side
+        action_layout = QVBoxLayout()
+        progress_layout = QHBoxLayout()
+        self._progress_widget = QWidget(self)
+        self._progress_filter = HoverEventFilter()
+        self._progress_bar = QProgressBar(self)
+        self._progress_widget.installEventFilter(self._progress_filter)
+        cancel_button = QPushButton()
+        cancel_button.setIcon(ima.icon('DialogCloseButton'))
+        cancel_button.hide()
+        progress_layout.addWidget(self._progress_bar)
+        progress_layout.addWidget(cancel_button)
+        self._progress_widget.setLayout(progress_layout)
+
+        self._progress_label = QLabel(_('Downloading'))
+        install_info = QLabel(
+            _('''Kite comes with a native app called the Copilot <br>'''
+              '''which provides you with real time <br>'''
+              '''documentation as you code.<br><br>'''
+              '''When Kite is done installing, the Copilot will <br>'''
+              '''launch automatically and guide you throught the <br>'''
+              '''rest of the setup process.'''))
+
+        button_layout = QHBoxLayout()
+        ok_button = QPushButton(_('OK'))
+        button_layout.addStretch()
+        button_layout.addWidget(ok_button)
+        button_layout.addStretch()
+
+        action_layout.addStretch()
+        action_layout.addWidget(self._progress_label)
+        action_layout.addWidget(self._progress_widget)
+        action_layout.addWidget(install_info)
+        action_layout.addStretch()
+        action_layout.addLayout(button_layout)
+
+        # Right side
+        copilot_image_source = get_image_path('kite_copilot.png')
+
+        copilot_image = QPixmap(copilot_image_source)
+        copilot_label = QLabel()
+        copilot_label.setPixmap(
```

Comment:
```suggestion        screen = QApplication.primaryScreen()
        copilot_image.setDevicePixelRatio(screen.devicePixelRatio())
        copilot_label.setPixmap(copilot_image)
```

Reply:
I attach the image again

![image](https://user-images.githubusercontent.com/3627835/66204745-294b3b80-e671-11e9-8788-f992f1e667d5.png)